app/pages/AppContainer.js

```diff
@@ -61,33 +62,36 @@ export class UnconnectedAppContainer extends Component {
 
   render() {
     const { isStaff, language } = this.props;
+    const mainContentId = 'main-content';
+
     return (
-      <div className={classNames('app', getCustomizationClassName())}>
-        <Helmet>
-          <title>Varaamo</title>
-        </Helmet>
+      <>
+        <AccessibilityShortcuts mainContentId={mainContentId} />
```

Comment:
This belong to main content, it should wrap the main content or being on a same level with main content, not above app wrap

Reply:
This link is something we should focus first when we tab into the page from the URL bar. From that POV I think it's the safest to have it as high on the page as possible. The original audit flag suggested vero.fi (https://www.vero.fi/) as an example for implementation and they, for instance, have the shortcuts on the next level from body.

Having it within the app wrapper doesn't communicate this fact with the same "intensity".

Saying this, if you still prefer to have it within the app wrapper, I'll move it.

Reply:
If its better to be on top, then it should be there, just my suggestion to keep things with same scope together